docs/operate/mobility/use-input-to-act.md

```diff
@@ -5,5 +5,105 @@ weight: 70
 layout: "docs"
 type: "docs"
 no_list: true
-description: "TODO"
+description: "Actuate your machine based on sensor readings or other inputs."
 ---
+
+You can program your machine to move based on sensor readings or other inputs.
+
+{{% alert title="Disambiguation" color="tip" %}}
+If you want to act or send alerts based on computer vision, see [Act based on inferences](/data-ai/ai/act/) or [Alert on inferences](/data-ai/ai/alert/).
+To alert based on data, see [Alert on data](/data-ai/data/advanced/alert-data/).
+{{% /alert %}}
```

Comment:
I think this info fits more naturally with the Get an input step - where you have some of it already, That gets rid of the alert here too...